### DISCUSSION TITLE
DateEdit changed on enter

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#3394] [Q&A]
Hey,

is it possible to only call the DateChanged EventCallback on the DateEdit, after the user presses enter. (Only if he types it manually)

Kind regards
Sebastian

**Comment by stsrki** [comment]
No, it is not possible. DateEdit is based on native `input type="date"` so it is controlled by the browser, and the browser will call `onchange` every time a part of value is changed.

**Reply by bastigerner** [reply]
Okay I understand... that is kind of sad... because I compare two dates and the comparison shouldnt start on input. It then instantly compares the year with for example 2 while somebody tries to type 2022.
